docker-compose.yaml

```diff
@@ -25,6 +7,7 @@ x-airflow-common:
     - ./services/etl/dags:/opt/airflow/dags
     - ./services/etl/logs:/opt/airflow/logs
     - ./services/etl/plugins:/opt/airflow/plugins
+    - ./services/airflow.cfg:/opt/airflow/airflow.cfg
```

Comment:
this is prop fine if it works, but seems a little strange to use a file as a volume

Reply:
roger, yes will try removing